components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/builder/AuthReqBuilder.java

```diff
@@ -234,50 +148,16 @@ public static AuthnRequest setSignature(AuthnRequest authnRequest, String signat
             
             Signer.signObjects(signatureList);
             return authnRequest;
-
-        } catch (Exception e) {
-            throw new Exception("Error while signing the SAML Request message", e);
+        } catch (CertificateEncodingException e) {
```

Comment:
Fixed in #1528 (https://github.com/wso2/carbon-apimgt/commit/c6d6cb237bd7654eda952fb2fbd74863e9899cfc)